build/script/init_agent

```diff
@@ -53,15 +54,25 @@ CLS_TO_POLICY_OFPORT=202
 CLS_TO_UPLINK_OFPORT=301
 UPLINK_TO_CLS_OFPORT=302
 
+LOCAL_GATEWAY_OFPORT=10
+UPLINK_GATEWAY_OFPORT=10
```

Comment:
Will this be used? Uplink Gateway ofport the same as Local Gateway ofport?

Reply:
They are not in same bridge, so it could be the same but not necessary.

Reply:
The UPLINK_GATEWAY_OFPORT defined but not used.

Reply:
Revised! L105 should be UPLINK_GATEWAY_OFPORT